tools/SetupFlow/DevHome.SetupFlow.UnitTest/TestingScenarios/Quickstart.md

```diff
@@ -0,0 +1,36 @@
+# Quickstart Playground Tests
```

Comment:
The script that creates the testing scenarios spreadsheet would be confused by this format. (Right?)
@EricJohnson327 Does the script only collect files linked to in the central list? If yes, this format is fine but it would be nice to have a linked file that just had the two test cases listed, so they'd be included in the spreadsheet.
If no, we need to change the format of this file.

Reply:
Ah, I didn't know about the script. Running it now, the .xlsx doesn't look right (each step is a different line in the file). Thanks for pointing this out.

I wasn't really aware of the 'schema' these tests should follow (I was sort of trying to copy the look-and-feel of other adjacent scenarios in the same folder). Is there a template I should have started from?

If you immediately know what the formatting problem is, let me know- I'm otherwise going to look at the .ps1 and figure out what it's expecting.

Reply:
Ok, I think I better understand the expected format. I've restructured the file so that the script should only extract the test case descriptions without picking up any of the extraneous instructions. I ran the script and I think that section looks like what I'd want to see.